docs/index.rst

```diff
@@ -112,9 +112,8 @@ Table of Contents
    :caption: API Basics
 
    tutorials/api-clusters
+   tutorials/api-clusters-adv
```

Comment:
Maybe a nit but I think the name `api-clusters-advanced` would be a bit more clear as a URL and filenames